.gitignore

```diff
@@ -50,22 +50,23 @@ env
 venv*
 env_py3
 envpy3
-env36
 local_config.py
 /superset_config.py
 /superset_text.yml
 superset.egg-info/
 superset/bin/supersetc
 tmp
 rat-results.txt
-superset/app/
```

Comment:
This is the incorrect path that contained `uploads/`. `superset/app/` shouldn't be a valid directory anyway. For developers, this change could potentially show up in their git repo as untracked files.